### PR TITLE
[rocksplicator][kafkaconsumer]: Allow kafka client parameters to be provided in configuration file

### PR DESCRIPTION
**Pull request by gopalrajpurohit**
Depends on : https://github.com/pinterest/rocksplicator/pull/368 pull request

### CODE REVIEW

File: common/kafka/kafka_consumer.cpp [hunk 1]
```diff
@@ -71,30 +73,106 @@ bool KafkaSeekWithRetry(RdKafka::KafkaConsumer* consumer,
...
+  kafkaConfigMap["enable.auto.offset.store"] = std::make_pair(
+    FLAGS_enable_kafka_auto_offset_store, false);
+
+  kafka::ConfigMap configMap;
+  if (!FLAGS_kafka_client_global_config_file.empty()) {
+    if (!kafka::KafkaConfig::read_conf_file(
+      FLAGS_kafka_client_global_config_file, &configMap)) {
+      LOG(ERROR) << "Can not read / parse config file: "
+                 << FLAGS_kafka_client_global_config_file
+                 << std::endl;
```
Comment (rajathprasad) [inline common/kafka/kafka_consumer.cpp:149]: Can you also increment the counter for for ErrorInit?

Comment (gopalrajpurohit) [reply common/kafka/kafka_consumer.cpp:149]: common::Stats::get()->Incr(getFullStatsName(
          kKafkaConsumerErrorInit, {"kafka_consumer_type=" + kafka_consumer_type}));
?? You meant stats?

Comment (rajathprasad) [reply common/kafka/kafka_consumer.cpp:149]: Yes, stats counter :)

**Comment by gopalrajpurohit** [comment]
Travis build failed with thrift_router which is known to be flaky. Rest of test passes.

**Pull request merged**